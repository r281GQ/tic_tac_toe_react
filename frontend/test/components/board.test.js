import React from 'react';
import { shallow } from 'enzyme';
import Board from './../../src/components/board';

describe('calorie_target_result_sum_label', () => {
  it('should render', () => {
    const component = shallow(
      <Board
        board={{
          1: '',
          2: '',
          3: '',
          4: '',
          5: '',
          6: '',
          7: '',
          8: '',
          9: ''
        }}
        makeHumanMove={value => value}
        playerSign="X"
        computerSign="O"
      />
    );
    expect(component).toHaveLength(1);
  });
});
