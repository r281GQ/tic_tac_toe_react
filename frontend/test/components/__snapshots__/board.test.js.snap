// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`calorie_target_result_sum_label should render 1`] = `
<div
  className="board__vertical__container"
>
  <div
    className="board__horizontal__container"
  >
    <div
      className="main__item__container"
      onClick="1"
    >
      <div
        className="main__item__paper"
      />
    </div>
    <div
      className="main__item__container"
      onClick="2"
    >
      <div
        className="main__item__paper"
      />
    </div>
    <div
      className="main__item__container"
      onClick="3"
    >
      <div
        className="main__item__paper"
      />
    </div>
    <div
      className="main__item__container"
      onClick="4"
    >
      <div
        className="main__item__paper"
      />
    </div>
    <div
      className="main__item__container"
      onClick="5"
    >
      <div
        className="main__item__paper"
      />
    </div>
    <div
      className="main__item__container"
      onClick="6"
    >
      <div
        className="main__item__paper"
      />
    </div>
    <div
      className="main__item__container"
      onClick="7"
    >
      <div
        className="main__item__paper"
      />
    </div>
    <div
      className="main__item__container"
      onClick="8"
    >
      <div
        className="main__item__paper"
      />
    </div>
    <div
      className="main__item__container"
      onClick="9"
    >
      <div
        className="main__item__paper"
      />
    </div>
  </div>
</div>
`;
